Alarmify/Modules/Edit/SubViews/FeaturesListVIew.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FeaturesListVIew.swift
+//  Alarmify
+//
+//  Created by Swastik Patil on 04/06/24.
+//
+
+import SwiftUI
+import AudioToolbox
+
+struct FeaturesListVIew: View {
+    @EnvironmentObject var alarmdata: AlarmData
+    var body: some View {
+        NavigationStack {
+            List {
+                DaySelectionView()
+                HStack {
+                    Text("Label")
+                    TextField("Silent alarm", text: $alarmdata.Alarmlabel)
+                        .multilineTextAlignment(.trailing)
+                }
+                SoundSelectorView()
+                Toggle("Vibrate", isOn: $alarmdata.vibrate).tint(.accentColor)
+                
+            }.preferredColorScheme(.dark)
```

Comment:
preferredColorScheme(.dark) - are you sure? App should support all themes. For testing, you can add the modifier to the Preview instead of adding it to the View. Same for every `preferredColorScheme` in the project.